### PR TITLE
Trimming now optional

### PR DESCRIPTION
**Pull request by sschmeier**
Ok, I think this does the job.

Let me know what you think.

Seb

### CODE REVIEW

File: .test/config.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 samples: samples.tsv
 units: units.tsv
 
+# trimming optional: yes or no
+trim: yes
```
Comment (johanneskoester) [inline .test/config.yaml]: I suggest:

```suggestion
adapter:
  trim: true
  sequence: ACGGATCGATCGATCGATCGAT
```

Comment (johanneskoester) [reply .test/config.yaml]: On the other hand, trimming might also involve non adapter sequences, right? E.g. quality trimming, depending on how people configure cutadapt. So maybe rather:

```suggestion
trimming:
  skip: false
  adapter: ACGGATCGATCGATCGATCGAT
```

Comment (sschmeier) [reply .test/config.yaml]: ok new merge request now

**Comment by johanneskoester** [comment]
Thank you!

**Pull request merged**